{"name": "test_on_second", "status": "broken", "statusDetails": {"message": "TypeError: BrowserType.launch() got an unexpected keyword argument 'logger'", "trace": "self = <test_oncall.TestOnCallTestKeywords object at 0x0000015270B37100>\n\n    @pytest.fixture()\n    def browser_setup(self):\n>       self.browser_on_call=self.OnCallFunctions.create_browser()\n\ntests\\OnCall\\test_oncall.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <controller.OnCallFunctions.OnCallFunctions object at 0x00000152708CBCB0>\n\n        def create_browser(self):\n            self.start()\n>           browser = self.pw.chromium.launch(headless=False,logger={\n            \"isEnabled\": lambda name, severity: name == \"api\",\n            \"log\": lambda name, severity, message, args: print(f\"{name}: {message}\")\n        }\n    )\nE   TypeError: BrowserType.launch() got an unexpected keyword argument 'logger'\n\ncontroller\\base.py:19: TypeError"}, "start": 1748332746295, "stop": 1748332746295, "uuid": "c81b8f35-355b-41cd-9f38-30e9e8a06413", "historyId": "2305e9176097a8930349d8331583c7b6", "testCaseId": "2305e9176097a8930349d8331583c7b6", "fullName": "tests.OnCall.test_oncall.TestOnCallTestKeywords#test_on_second", "labels": [{"name": "story", "value": "Second"}, {"name": "tag", "value": "Second"}, {"name": "tag", "value": "critical"}, {"name": "feature", "value": "OnCall"}, {"name": "parentSuite", "value": "tests.OnCall"}, {"name": "suite", "value": "test_oncall"}, {"name": "subSuite", "value": "TestOnCallTestKeywords"}, {"name": "host", "value": "DESKTOP-G02SEED"}, {"name": "thread", "value": "24136-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.OnCall.test_oncall"}]}